NuGet/OpenRiaServices.Client.CodeGen/build/OpenRiaServices.CodeGen.targets

```diff
@@ -354,7 +352,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
        UseFullTypeNames="$(OpenRiaClientUseFullTypeNames)"
        ClientAssemblySearchPaths="@(OpenRiaClientCodeGenClientAssemblySearchPath)"
        CodeGeneratorName="$(OpenRiaClientCodeGeneratorName)"
-       CopyOrLinkSharedFiles="Link">
+       SharedFilesMode="$(OpenRiaSharedFilesMode)">
```

Comment:
wouldnt it be more intuitive if the user had to specify LinkSharedFilesInsteadOfCopy (bool) instead of a Link/Copy enum? 

e.g. do we expect there to be other options than Link vs Copy in the future?

Reply:
I don't expect any other alternatives in the near future, except maybe a CopyAndLink option which gives all the pros with the new solution but which still copies the files.

I've updated the documentation to say that the setting is a temporary backwards compatibility switch and that it will probably be removed in the future (as long as there are not any major issues with the new alternative)